assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * DashboardAllTrafficWidget component
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Widgets from 'googlesitekit-widgets';
+import { STORE_NAME as MODULES_ANALYTICS, FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
+import { STORE_NAME as CORE_FORMS } from '../../../../../googlesitekit/datastore/forms/constants';
+import whenActive from '../../../../../util/when-active';
+import TotalUserCount from './TotalUserCount';
+import UserCountGraph from './UserCountGraph';
+import DimensionTabs from './DimensionTabs';
+import UserDimensionsPieChart from './UserDimensionsPieChart';
+import SourceLink from '../../../../../components/SourceLink';
+import { Grid, Row, Cell } from '../../../../../material-components/layout';
+const { Widget } = Widgets.components;
+const { useSelect } = Data;
+
+function DashboardAllTrafficWidget() {
+	const dimensionName = useSelect( ( select ) => select( CORE_FORMS ).getValue( FORM_ALL_TRAFFIC_WIDGET, 'dimensionName' ) || 'ga:channelGrouping' );
+	const dimensionValue = useSelect( ( select ) => select( CORE_FORMS ).getValue( FORM_ALL_TRAFFIC_WIDGET, 'dimensionValue' ) );
+	const serviceReportURL = useSelect( ( select ) => select( MODULES_ANALYTICS ).getServiceReportURL( 'trafficsources-overview' ) );
+
+	return (
+		<Widget
+			slug="analyticsAllTrafficV2"
+			footer={ () => (
+				<SourceLink
+					className="googlesitekit-data-block__source"
+					name={ _x( 'Analytics', 'Service name', 'google-site-kit' ) }
+					href={ serviceReportURL }
+					external
+				/>
+			) }
```

Comment:
This is not matching what's described in the ACs / displayed in the Figma file: The source link should appear in the lower right. The `footer` prop renders these in our standard way (which we'll probably change in the future), but we'll need to adjust for the design requirements manually here:
* Either we don't use the prop and render the `SourceLink` manually within an extra `Row` and some styling.
* Or we add custom CSS rules for this specific widget footer.

Reply:
@felixarntz how about we leave it like this for this issue. As I see there is #2428 around this issue and maybe we can add the designing part to that instead. 
I am only suggesting as this issue is blocking 5 other tickets for quite some time.

Reply:
I've fixed this in https://github.com/google/site-kit-wp/pull/2512/commits/a48f812cdaa493b2ed34ff97d856e7c1f034455f :)